app/assets/stylesheets/views/_search.scss

```diff
@@ -133,6 +133,24 @@ main.search {
         float: left;
         width: $two-thirds;
       }
+
+      mark {
+        background-color: rgba(255, 255, 140, 0.5);
```

Comment:
I wonder if it would be a problem in the real world. Perhaps we could see how many of the top X search terms match the page title exactly.

I don't think it's a problem when it's a partial match.

The other solution is not highlight the title when it is a complete match.